github/blind.css

```diff
@@ -1,3 +1,6 @@
+body.br-blinded .br-label::after {
+  content: attr(redacted-label) !important;
```

Comment:
It took me a while to understand this is actually changing the content of the final tooltip element.  Almost too clever. ;)